frontend/src/global_styles/content/work_packages/shared/_file_list.sass

```diff
@@ -76,6 +76,13 @@
       &-icon
         margin-right: $spot-spacing-0_25
 
+    & &-button
```

Comment:
I don't think you need the double & here, it just adds an extra class of specificity

Reply:
yep, without it, the styles from `.spot-link` are loaded on top of this and thus `display: inline` overwrites the `display: flex`

Reply:
you think it is okay otherwise?